bouncer/app/models/bootstrap.py

```diff
@@ -41,3 +41,10 @@ def insert_bootstrap_data(dbsession, bouncer_config):
         provider_id=None,
         )
     )
+
+
+def tables_cleanup_order(tables):
```

Comment:
Is this still WIP?

Reply:
It is not wip, the upstream bouncer overrides this with `ace` and `user_groups` tables.

Reply:
Aaah! Makes sense.